### ISSUE TITLE
403 on curl -I https://get.docker.com/builds/Linux/x86_64/docker-1.13.0.tgz

### ISSUE THREAD

**Issue by dpacaud** [docker/for-linux#470]
* [x] This is a bug report
* [ ] This is a feature request
* [x] I searched existing issues before opening this one

### Expected behavior

`curl -I "https://get.docker.com/builds/Linux/x86_64/docker-1.13.0.tgz"`

Should give us a 200 and the tar

### Actual behavior

`curl -I "https://get.docker.com/builds/Linux/x86_64/docker-1.13.0.tgz"`

Gives us 403 forbidden. I have tried this locally, on a different network, and on AWS EU-WEST-1
We are based in france

### Steps to reproduce the behavior

N/A

**Output of `docker version`:**

```
N/A
```

**Output of `docker info`:**

```
N/A
```

**Additional environment details (AWS, VirtualBox, physical, etc.)**

AWS and local network (France)

**Comment by seemethere** [comment]
Yeah that's really strange, I'm getting a `200` in the US but a `403` in `FR`

<img width="1389" alt="screen shot 2018-10-29 at 11 48 47 am" src="https://user-images.githubusercontent.com/1700823/47673025-aff9bd80-db70-11e8-92b3-8da7d0fccc8e.png">

**Comment by tspadi** [comment]
It seems like the endpoint isn't available anymore, neither from Europe nor US now ...

**Comment by dangreencorn** [comment]
This is also an issue for docker-1.11.2.tgz in us-west-2

**Comment by dpacaud** [comment]
FYI, it seems that it works if you bypass cloudfront and go directly to the S3 bucket (http://get.docker.com.s3.amazonaws.com/builds/Linux/x86_64/docker-1.10.3.tgz)
There is however no way for us to make sure that this is actually the endpoint behind https://get.docker.com/builds/Linux/x86_64/docker-1.13.0.tgz so this workaround might be a security issue.

**Comment by seemethere** [comment]
@dpacaud Should be resolved now, can you try with the original link?
